tests/unit/lms/services/async_oauth_http_test.py

```diff
@@ -0,0 +1,81 @@
+import asyncio
+from unittest.mock import sentinel
+
+import httpretty
+import pytest
+from aiohttp import TooManyRedirects
+from aioresponses import CallbackResult, aioresponses
+
+from lms.services import ExternalRequestError
+from lms.services.async_oauth_http import AsyncOAuthHTTPService, factory
+
+
+@pytest.mark.filterwarnings("ignore::RuntimeWarning")
+class TestAsyncOAuthHTTPService:
```

Comment:
These two test are not asserting much or anything about the async details of the service.